doc/connectivity/connectome_extraction.rst

```diff
@@ -0,0 +1,232 @@
+.. _connectome_extraction:
+
+============================================================================
+Connectome extraction: inverse covariance for direct connections 
+============================================================================
+
+.. topic:: **Page summary**
+
+   Given a set of time-series (eg as extracted in the previous section)
+   A *functional connectome* is a set of connections representing brain
+   interactions between regions. Here we show the use of sparse-inverse
+   covariance to extract functional connectomes focussing only on direct
+   interactions between regions.
+
+.. contents:: **Contents**
+    :local:
+    :depth: 1
+
+.. topic:: **References**
+
+   * `Smith et al, Network modelling methods for FMRI,
+     NeuroImage 2011 <http://www.sciencedirect.com/science/article/pii/S1053811910011602>`_
+
+   * `Varoquaux and Craddock, Learning and comparing functional
+     connectomes across subjects, NeuroImage 2013
+     <http://www.sciencedirect.com/science/article/pii/S1053811913003340>`_ 
+
+Sparse inverse covariance for functional connectomes
+=====================================================
+
+Resting-state functional connectivity can be obtained by estimating a
+covariance (or correlation) matrix for signals from different brain
+regions. The same information can be represented as a weighted graph,
+vertices being brain regions, weights on edges being covariances
+(gaussian graphical model). However, coefficients in a covariance matrix
```

Comment:
- mention that non-directional
- mention what is "sparse" about it
- in direct connections (i.e., influence by third-party sources)